lib/coprl/presenters/plugins/theme.rb

```diff
@@ -97,7 +22,16 @@ def view_dir_theme(_pom)
           # It will be called once for the page.
           # The pom is passed along with the sinatra render method.
           def render_header_theme(pom, render:)
-            render.call :erb, 'theme_header', views: view_dir_theme(pom)
+            theme = Theme::Selector.call(context: pom.send(:context))
+            render.call :erb, 'theme_header', views: view_dir_theme(pom),
+                        locals: { theme: theme, custom_font_family: custom_font_family(theme) }
+          end
+
+          def custom_font_family(theme)
+            return unless theme&.font_family&.present?
+            return unless theme.font_family != Palette::FONTS[:default]
```

Comment:
minor nit: the double negative's a bit hard to grok at first.
```suggestion
            return if theme.font_family == Palette::FONTS[:default]
```

Reply:
Will make changes here in a bit